setup/webmail.sh

```diff
@@ -122,34 +125,52 @@ cat > $RCM_CONFIG <<EOF;
 \$config['support_url'] = 'https://mailinabox.email/';
 \$config['product_name'] = '$PRIMARY_HOSTNAME Webmail';
 \$config['des_key'] = '$SECRET_KEY';
-\$config['plugins'] = array('html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login', 'carddav');
-\$config['skin'] = 'larry';
-\$config['login_autocomplete'] = 2;
-\$config['password_charset'] = 'UTF-8';
-\$config['junk_mbox'] = 'Spam';
-?>
 EOF
 
+if [ "${DISABLE_NEXTCLOUD}" == "0" ]; then
+	cat >> $RCM_CONFIG <<EOF;
+	\$config['plugins'] = array('html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login');
+	\$config['skin'] = 'larry';
+	\$config['login_autocomplete'] = 2;
+	\$config['password_charset'] = 'UTF-8';
+	\$config['junk_mbox'] = 'Spam';
+	?>
+	EOF
+else
+	cat >> $RCM_CONFIG <<EOF;
+	\$config['plugins'] = array('html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login', 'carddav');
+	\$config['skin'] = 'larry';
+	\$config['login_autocomplete'] = 2;
+	\$config['password_charset'] = 'UTF-8';
+	\$config['junk_mbox'] = 'Spam';
+	?>
+	EOF
+fi
+	
```

Comment:
There seems to be a lot of duplicated code here so this can probably be simplified.  It looks like the only difference is "carddav".